src/Shell.tsx

```diff
@@ -89,16 +107,27 @@ class ShellContainer extends React.Component<Props, {}> implements ShellFunction
             'wc-mic',
             !showMicButton && 'hidden',
             this.props.listening && 'active',
-            !this.props.listening && 'inactive',
+            !this.props.listening && 'inactive'
         );
 
         return (
-            <div className={className}>
-                <input id="wc-upload-input" type="file" ref={ input => this.fileInput = input } multiple onChange={ () => this.onChangeFile() } />
-                <label className="wc-upload" htmlFor="wc-upload-input">
+            <div className={ className }>
+                <label
```

Comment:
We have to move DOM around to show the focus indicator correctly. In short, `<label>` should wrap around `<input>` so we don't need `htmlFor` prop and ID-ing everything.

Reply:
OIC. The label trick gives us a free "click" event on the file uploader.

Reply:
Yes, until we figured out the "free click" does not works with "spacebar" or "enter". :(